better_output.md

```diff
@@ -0,0 +1,44 @@
+# Better Outputs
```

Comment:
drop this file and update the documentation. Create an ASCIInema showing the usage of the script